runtime/vimrc_example.vim

```diff
@@ -62,26 +40,15 @@ if has("autocmd")
   " position when opening a file.
```

Comment:
Can't comment on it, but maybe remove the line 3 lines above? The one regarding gvim, given nvim is all about unifying interfaces and whatnot.